lib/gui/app.js

```diff
@@ -86,23 +91,40 @@ app.run(() => {
 app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateModel) => {
   AnalyticsService.logEvent('Application start');
 
-  const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates();
+  const currentVersion = packageJSON.version;
+  const currentReleaseType = release.getReleaseType(currentVersion);
+  const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates({
+    ignoreSleepUpdateCheck: currentReleaseType !== release.RELEASE_TYPE.PRODUCTION
+  });
 
-  if (!shouldCheckForUpdates || process.env.ETCHER_DISABLE_UPDATES) {
+  if (_.some([
+    !shouldCheckForUpdates,
+    process.env.ETCHER_DISABLE_UPDATES,
+    currentReleaseType === release.RELEASE_TYPE.UNKNOWN
+  ])) {
     AnalyticsService.logEvent('Not checking for updates', {
       shouldCheckForUpdates,
-      disableUpdatesEnvironmentVariable: process.env.ETCHER_DISABLE_UPDATES
+      disableUpdatesEnvironmentVariable: process.env.ETCHER_DISABLE_UPDATES,
+      releaseType: currentReleaseType
     });
     return;
   }
 
+  const updateSemverRange = packageJSON.updates.semverRange;
+  const useUnstableChannel = settings.get('useUnstableUpdateChannel');
```

Comment:
Hmmm, I wonder if it'd make sense to rename this (both here, and everywhere else it's used) from `useUnstableChannel` to `includeUnstableChannel`? That way it might be more obvious that 'unstable' is a superset of 'stable' (i.e. being on the unstable channel might still get you newer stable versions), and not a distinct set?